### PR TITLE
#157930463 Add endpoint functionality for URL /api/v1/register

### PR DESCRIPTION
**Pull request by artorious**
#### What does this PR do?

Add endpoint functionality /api/v1/register

#### Description of Task to be completed?

* Version API endpoint (route)

* Write tests for the endpoint 

* Build functionality to pass tests

* Test endpoint on postman

#### How the app should be tested
On a webbrowser:

On a  terminal (Linux):

* install am configure [pytest](https://docs.pytest.org/en/latest/), [virtualenv](http://docs.python-guide.org/en/latest/dev/virtualenvs/) and [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/install.html)

* Install and configure [postman](https://www.getpostman.com/docs/v6/postman/launching_postman/installation_and_updates#linux)

* Install and set up git `sudo apt install git` 

* Clone the repository `git clone  https://github.com/artorious/MaintenanceTracker.git`

* Run `git checkout ft-user-registration-157930463`

* Run `cd MaintenanceTracker`

* Run `pip install requirements.txt`

* Run `pytest app/tests/*`

* Run `export FLASK_APP=run.py`

* Run `flask run` to launch the flask server

* on postman; POST /api/v1/register

#### What are the relevant pivotal tracker stories?
[#157930463](https://www.pivotaltracker.com/story/show/157930463)

#### Questions:

### CODE REVIEW

**Comment by Slinjez** [comment]
@artorious , I hope you don't mind sharing screenshots... it's easier and more time saving

**Comment by artorious** [comment]
@Slinjez Done. Thanks

**Comment by Slinjez** [comment]
@artorious I see it gave feedback... ok,.. its cool man

File: app/views.py [hunk 1]
```diff
@@ -3,8 +3,26 @@
 from app.models import Users
 from app import app
 
+# Init
+sample_account = Users()
+
 
 @app.route('/api/v1/register', methods=['POST'])
 def register():
```
Comment (daisymacharia) [inline app/views.py:11]: the function doesn't check whether the user has inputted any data in their payload before they make the post, also doesnt check whether the user already exists before they can be appended to the dictionary

Comment (artorious) [reply app/views.py:11]: Will do

**Pull request merged**